src/components/Chart.spec.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { shallow } from "enzyme";
```

Comment:
This all looks good. I'm curious what the benefit of using Enzyme to Jest as far as unit testing. I definitely have just scratched the surface as to the benefits of unit testing, so I'm not pushing one or the other.